### PR TITLE
Update readme for bindings v0.0.8

### PR DESCRIPTION
**Pull request by nighthawk9**
I would also appreciate suggestions on changing the organization of the new keybindings.

### CODE REVIEW

File: README.org [hunk 1]
```diff
@@ -27,14 +27,37 @@ will setup the new key bindings for you.
...
+| Change sorting***               | o         | O           |
+| Rerun search                    | gr        |             |
+| Toggle include related          | zr        |             |
+| Toggle threading                | zt        |             |
+| Toggle hide cited               | za        |             |
+| Skip duplicates                 | zd        |             |
+| Show log                        | gl        |             |
+| Select other view               | gv        |             |
+| Save attachement                | p         |             |
+| Save multiple attachments       | P         |             |
```
Comment (Ambrevar) [inline README.org]: You can remove this line, from mu4e 1.0 onward the two functions are an alias.

**Comment by Ambrevar** [comment]
What do you mean with "organization"?

I suggest we also add `yu` as `mu4e-view-save-url`.

The table in the comment section of the file needs to be updated as well.

**Comment by nighthawk9** [comment]
Updated the comment section of the elisp code. Added the save URL to both the readme and the elisp code.

What I mean by organization is that currently the key bindings do not stay constant from the main mode map -> header mode map -> view mode map. This leaves me with the thought of: should we wait for the rest of the keybindings to be fleshed out? Or make three separate tables for the keybindings? 

As of right now, I don't think the "noting" solution is as clean as it could be.

I am relatively new to the extending features of git (rebasing, squashing, bisecting and cherry picking) to have the cleanest commit log for a branch, as well as contributing to open source projects. I thought I would start contributing to projects I use often to learn the ropes.

I appreciate the help.

**Comment by Ambrevar** [comment]
Well, it's 3 different modes with different bindings, with a bunch in common.
The more consistent the better.  I'm quite happy with the current bindings, but if you think there is anything we could improve, don't hesitate to send a pull request.

As for git, a good history is a history made of atomic commits: one change one commit.
In this case, it's simple enough that all commits can be squashed into just one.
You can use the "rebase interactively" feature of git and then "squash" or "fixup" all commits but the first.

I also recommend Magit if you are not using it already.

**Comment by Ambrevar** [comment]
@nighthawk9: Let me know if you want to rebase or if I should squash your commits and merge.

**Comment by nighthawk9** [comment]
Feel free to squash the commits and merge.

I appreciate your help. Thank you.

**Pull request merged**